### PR TITLE
Db queries file

### PR DESCRIPTION
**Pull request by hmalmsten**

### CODE REVIEW

**Review by jniemii — APPROVED** [review]
The return value of executeQuery probably isn't correct but we can fix it later

File: backend/src/db/queries.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import { logger } from '../utils/logger';
+
+const executeQuery = async (
+    text: string,
+    values: string[]
+): Promise<string[]> => {
+    const query = {
+        text: text,
+        values: values,
+    };
```
Comment (itulkki) [inline backend/src/db/queries.ts]: this could be written simply as
const query = {text, values};

**Review by itulkki — DISMISSED** [review]
Please add some comments explaining what happens if the queries are unsuccessful in some way

File: backend/src/db/queries.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    await pool.connect();
+    const res = await pool.query(query);
+    logger.info(res);
+    await pool.end();
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+    return res.rows;
+};
+
+const selectProjectsbyUserID = async (userID: string) => {
```
Comment (jaakkoha-bytecraft) [inline backend/src/db/queries.ts]: These should be `export const ...` and contain a return type, since right now all of these will return `any`.

**Comment by hmalmsten** [comment]
> Please add some comments explaining what happens if the queries are unsuccessful in some way

It seems @jniemii has error handling for this resolved in a child branch

**Review by jaakkoha-bytecraft — APPROVED** [review]

**Pull request merged**